plugins/node/instrumentation-amqplib/src/amqplib.ts

```diff
@@ -74,6 +74,8 @@ import {
 } from './utils';
 import { VERSION } from './version';
 
+const supportedVersions = ['>=0.5.5 <1'];
```

Comment:
nit: Perhaps this is academic because there isn't an amqplib 1.x release, and I'm guessing there isn't one in the short term. However, this is a "docs:" PR that changes instr-amqplib code. Perhaps mention this change is adding an upper limit for this instr?

Reply:
I allowed myself to do that, because I was looking at the current major and capped it there. While very unlikely that any user will be affected by this, I don't mind documenting it just in case. Do you think the scope should change from `docs` to `fix!`?

Reply:
Can the scope be `docs, chore: ...`?  I suppose it could be `fix` as well. I don't think it is realistically a breaking change for any of the instrumentations changed, but I'm not opposed to a very conservative `fix!`.

Reply:
FWIW I prefer to err on the side of conservative, and point out _possible_ behavior changes even if they are unlikely. Renaming from `docs` to`chore` could be sufficient here (...depending on the definition of `chore`, which differs across resources as it could be "non-source changes" - disqualifying this - or "routine tasks" - where this fits... but that might be too far off-course to have that convo here 😅 )